apiml-security-common/src/test/java/org/zowe/apiml/security/common/verify/TrustedCertificatesProviderTest.java

```diff
@@ -158,13 +160,5 @@ void whenGetTrustedCerts_thenNoCertificatesReturned() {
             assertNotNull(result);
             assertTrue(result.isEmpty());
         }
-
```

Comment:
What's the reason for removing the test?

Reply:
Sonar had issue with duplicate test. We checked that with Pavel. The test is an exact duplicate of the one above. In history, they differed in mock and they mocked some api which was removed. The deprecated API mock was removed then which resulted in the tests being the same.

Reply:
ok thanks for the clarification. The rest looks good to me, I will approve it once the build is green.